CMakeLists.txt

```diff
@@ -248,11 +250,6 @@ if(CMAKE_BUILD_TYPE STREQUAL "Coverage")
     endif(CMAKE_COMPILER_IS_GNU OR CMAKE_COMPILER_IS_CLANG)
 endif(CMAKE_BUILD_TYPE STREQUAL "Coverage")
 
-if(LIB_INSTALL_DIR)
-else()
-    set(LIB_INSTALL_DIR lib)
-endif()
-
```

Comment:
For backward compatibility `CMAKE_INSTALL_LIBDIR` must be set if `LIB_INSTALL_DIR` is set.

Reply:
Done. Please review.